File: activity.java
import java.util.ArrayList;
import java.util.Scanner;

class TaskScheduler {
    private static ArrayList<String> tasks = new ArrayList<>();

    // Function to add a task
    public static void addTask(String task) {
        tasks.add(task);
        System.out.println("Task added: " + task);
    }

    // Function to run the scheduler
    public static void runScheduler() {
        System.out.println("\nRunning Task Scheduler...");
        if (tasks.isEmpty()) {
            System.out.println("No tasks to run.");
        } else {
            for (String task : tasks) {
                System.out.println("Executing task: " + task);
            }
        }
    }

    // Main function
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        while (true) {
            System.out.println("\nTask Scheduler Menu:");
            System.out.println("1. Add Task");
            System.out.println("2. Run Scheduler");
            System.out.println("3. Exit");
            System.out.print("Enter your choice: ");

            int choice = scanner.nextInt();
            scanner.nextLine(); // Consume the newline character

            switch (choice) {
                case 1:
                    System.out.print("Enter task: ");
                    String task = scanner.nextLine();
                    addTask(task);
                    break;
                case 2:
                    runScheduler();
                    break;
                case 3:
                    System.out.println("Exiting Task Scheduler...");
                    scanner.close();
                    System.exit(0);
                default:
                    System.out.println("Invalid choice. Please try again.");
            }
        }
    }
}

File: activity.py
# Initialize an empty list to store tasks
tasks = []

# Function to add a task
def add_task():
    task = input("Enter task: ")
    tasks.append(task)
    print(f"Task added: {task}")

# Function to run the scheduler
def run_scheduler():
    print("\nRunning Task Scheduler...")
    if not tasks:
        print("No tasks to run.")
    else:
        for task in tasks:
            print(f"Executing task: {task}")

# Main function
def main():
    while True:
        print("\nTask Scheduler Menu:")
        print("1. Add Task")
        print("2. Run Scheduler")
        print("3. Exit")
        choice = input("Enter your choice: ")

        if choice == "1":
            add_task()
        elif choice == "2":
            run_scheduler()
        elif choice == "3":
            print("Exiting Task Scheduler...")
            break
        else:
            print("Invalid choice. Please try again.")

# Run the script
if __name__ == "__main__":
    main()
    
